### PR TITLE
Add startup time and boot_timestamp

### PR DESCRIPTION
**Pull request by ludeeus**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (which adds functionality to the supervisor)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request: https://github.com/home-assistant/developers.home-assistant/pull/1064
- Link to cli pull request:

## Checklist

- [x] The code change is tested and works locally.
- [ ] Local tests pass. **Your PR cannot be merged unless tests pass**
- [ ] There is no commented out code in this PR.
- [ ] I have followed the [development checklist][dev-checklist]
- [ ] The code has been formatted using Black (`black --fast supervisor tests`)
- [x] Tests have been added to verify that the new code works.

If API endpoints of add-on configuration are added/changed:

- [x] Documentation added/updated for [developers.home-assistant.io][docs-repository]

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[docs-repository]: https://github.com/home-assistant/developers.home-assistant

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ async def mock_get_properties(dbus_obj, interface):
         if latest.isnumeric():
             fixture = f"{fixture}_{latest}"
 
-        return load_json_fixture(f"{fixture}.json")
+        return load_json_fixture(f"{fixture or 'base'}.json")
```
Comment (pvizeli) [inline tests/conftest.py:75]: Why we need that?

Comment (ludeeus) [reply tests/conftest.py:75]: https://github.com/home-assistant/supervisor/pull/3136/files#diff-e891abd55fca9889061fdb3c890811f14179b0e4463bc0dbbd78e3f5a362091dR119
This is `""`
Which would look for  the file `fixtures/.json`

Comment (ludeeus) [reply tests/conftest.py:75]: Not without patching this helper in this single test

Comment (ludeeus) [reply tests/conftest.py:75]: We don't require 100% coverage, so can just remove the testing parts out of the PR :shrug:

**Review by pvizeli — APPROVED** [review]

**Pull request merged**